docs/src/use/getting-started.md

```diff
@@ -27,25 +27,13 @@ npm init @eslint/config
 If you want to use a specific shareable config that is hosted on npm, you can use the `--config` option and specify the package name:
 
 ```shell
-# use `eslint-config-semistandard` shared config
+# use `eslint-config-standard` shared config
 
 # npm 7+
-npm init @eslint/config -- --config semistandard
-
-# or (`eslint-config` prefix is optional)
-npm init @eslint/config -- --config eslint-config-semistandard
+npm init @eslint/config -- --config eslint-config-standard
 
 # ⚠️ npm 6.x no extra double-dash:
```

Comment:
Similar to https://github.com/eslint/create-config/pull/81, can we also remove npm 6.x specifics from this document?